app/Http/Controllers/Admin/ColumnMonsterCrudController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;
+
+/**
+ * Class ColumnMonsterCrudController.
+ *
+ * @property-read \Backpack\CRUD\app\Library\CrudPanel\CrudPanel $crud
+ */
+class ColumnMonsterCrudController extends MonsterCrudController
+{
+    /**
+     * Configure the CrudPanel object. Apply settings to all operations.
+     *
+     * @return void
+     */
+    public function setup()
+    {
+        CRUD::setModel(\App\Models\Monster::class);
+        CRUD::setRoute(config('backpack.base.route_prefix').'/column-monster');
+        CRUD::setEntityNameStrings('column monster', 'column monsters');
+        $this->crud->set('show.setFromDb', false);
+    }
+
+    /**
+     * Define what happens when the List operation is loaded.
+     *
+     * @see  https://backpackforlaravel.com/docs/crud-operation-list-entries
+     *
+     * @return void
+     */
+    public function setupListOperation()
+    {
+        $this->crud->disableResponsiveTable();
+
+        $timeSpaceColumns = static::getFieldsArrayForTimeAndSpaceTab();
+
+        foreach ($timeSpaceColumns as $columnKey => $column) {
+            // transform field array names into comma separated string
+            if (is_array($column['name'])) {
+                $timeSpaceColumns[$columnKey]['name'] = implode(',', $column['name']);
+            }
+        }
+
+        $this->crud->addColumns(static::getFieldsArrayForSimpleTab());
+        $this->crud->addColumns($timeSpaceColumns);
+        $this->crud->addColumns(static::getFieldsArrayForSelectsTab());
+        $this->crud->addColumns(static::getFieldsArrayForRelationshipsTab());
+        $this->crud->addColumns(static::getFieldsArrayForUploadsTab());
+        $this->crud->addColumns(static::getFieldsArrayForWysiwygEditorsTab());
+        $this->crud->addColumns(static::getFieldsArrayForMiscellaneousTab());
+
+        foreach ($this->crud->columns() as $columnKey => $column) {
```

Comment:
this needs to be done elsewhere

Reply:
I decided to open a different issue for this matter - see https://github.com/Laravel-Backpack/CRUD/issues/4945
I'm not sure we _should_ do anything, tbh.